partner-portal/src/API/Program.cs

```diff
@@ -68,12 +69,22 @@
         .RequireRole(Claims.IdentityProvider, Roles.Dmft));
 });
 
+// NOTE temporary logger code, replace after adding logger e.g. Serilog/Splunk
+using var loggerFactory = LoggerFactory.Create(loggingBuilder => loggingBuilder
```

Comment:
can you add a TODO comment here "remove loggerFactory.Create and get the loggerFactory from ".AddSerilogBootstrapLogger"
The reason being, is because we are creating a temporary logging factory, then adding a real logger, than using the temporary loggerFactory when we should just be using the logger we just created.